batch/processor_Block_Sync_Status.py

```diff
@@ -65,118 +123,138 @@ def append(self, data):
         self._buffer[self._next] = data
         self._next = (self._next + 1) % len(self._buffer)
 
-    def peekOldest(self):
+    def peek_oldest(self):
         return self._buffer[self._next]
 
 
-# Watcher
-class Watcher:
-    def __init__(self):
-        pass
+class Processor:
```

Comment:
Almost the same code.
https://github.com/BoostryJP/ibet-Prime/blob/dev-21.12/batch/processor_monitor_block_sync.py